internal/command/extensions/supabase/dashboard.go

```diff
@@ -43,7 +43,9 @@ func runDashboard(ctx context.Context) (err error) {
 		}
 
 		err = extensions_core.OpenDashboard(ctx, extension.Name)
-
+		if err != nil {
+			return err
+		}
```

Comment:
no need since https://github.com/superfly/flyctl/pull/2998

Reply:
Aah, missed that PR. I'll drop the commit.